service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ManageHearingsControllerEditHearingMidEventTest.java

```diff
@@ -42,6 +42,20 @@ void shouldPopulatePreviousVenueFieldsWhenUserSelectsAddNewHearing() {
         assertThat(updatedCaseData.getPreviousHearingVenue()).isEqualTo(PreviousHearingVenue.builder()
             .previousVenue("Aberdeen Tribunal Hearing Centre, 48 Huntly Street, AB1, Aberdeen, AB10 1SH")
             .build());
+        assertThat("Yes".equals(updatedCaseData.getHasPreviousHearingVenue()));
```

Comment:
normally assertions would be written like assertThat(thing).equals("Yes") (same on line 58)